README.md

```diff
@@ -45,6 +45,43 @@ This integration is included in HACS. Search for the `Somfy TaHoma` integration
 
 ## Advanced
 
+### Examples
+#### Using a template_cover
+If your cover supports the slow moving mode you can create template_covers to move the cover always in the slow mode. Add in `configuration.yaml` the following code, replacing the `cover.your_cover` with your entity name:
+```yml
+cover:
+  - platform: template
```

Comment:
I have never used a cover template, but don't we need to add more properties? (https://www.home-assistant.io/integrations/cover.template/#covers).

For example: `stop_cover` and `set_cover_position`?